assets/locales/en/translation.json

```diff
@@ -11,6 +11,7 @@
   "pa-IN": "Punjabi",
   "te": "Telugu",
   "pt-BR": "Portuguese (Brazil)",
+  "it":"Italiano",
```

Comment:
changing things here will have no effect. these values are copied over from appium desktop (it's weird, i know, and we should probably change the way this works). so you'll need to make a PR to appium desktop first with these changes.